src/utils/JSONHelper.ts

```diff
@@ -16,14 +17,14 @@ function replacer(this: any, k: string, v: any) {
 }
 
 export function stringify(input: any) {
-    return JSON.stringify(input, replacer);
+    return jsonStringify(input, replacer);
```

Comment:
Why are you changing this out? :thinking:

Reply:
What do you mean? The `jsonStringify` is from the SDK, which replaces the strong types from the SDK with wrapper versions, similar to what is done with `bigint` and `Date` here.